engines/sherlock/detection_tables.h

```diff
@@ -220,11 +220,11 @@ static const SherlockGameDescription gameDescriptions[] = {
 		// Provided by AquariumTroop
 		{
 			"scalpel",
-			_s("Missing game code"), // Reason for being unsupported
+			nullptr,
```

Comment:
We normally put `""` here

Reply:
In this file I see only:
			"scalpel",
			0,
Which one should I use: 0, nullptr or "" ?

Reply:
ah, okay, leave it as nullptr then